venues/ICLR.cc/2018/Conference/python/admin-init.py

```diff
@@ -39,6 +39,9 @@
 groups[config.REVIEWERS_DECLINED] = openreview.Group(config.REVIEWERS_DECLINED, **config.reviewer_group_params)
 groups[config.REVIEWERS_EMAILED] = openreview.Group(config.REVIEWERS_EMAILED, **config.reviewer_group_params)
 
+groups[config.REVIEWERS_PLUS] = openreview.Group(config.REVIEWERS_PLUS, members = [config.CONF, config.REVIEWERS, config.AREA_CHAIRS, config.PROGRAM_CHAIRS], **config.public_group_params)
+groups[config.AREA_CHAIRS_PLUS] = openreview.Group(config.AREA_CHAIRS_PLUS, members = [config.CONF, config.AREA_CHAIRS, config.PROGRAM_CHAIRS], **config.public_group_params)
```

Comment:
these are so that commenters can choose the reader of their comments: "reviewers_plus" is a group that represents reviewers, area chairs, and program chairs. "area_chairs_plus" is a group that represents just the area chairs and the program chairs.